Day 2. Homework

eclimov opened the pull request:
Entities created. 
+ minor security improvements

eclimov committed:
- ORM entities added: "Affiliate", "Category", "Job"
- Migrations generated and executed

eclimov committed:
- Not using hardcoded confidential values in "docker-compose.yml" anymore

gregurco commented on src/Entity/Affiliate.php:
```diff
@@ -0,0 +1,155 @@
+     * @var datetime
+     * @ORM\Column(type="datetime")
+     */
+    private $created_at;
```
camelCase

gregurco commented on src/Entity/Affiliate.php:
```diff
@@ -0,0 +1,155 @@
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Class Affiliate
```
remove this useless comment `Class Affiliate`

gregurco commented on src/Entity/Job.php:
```diff
@@ -0,0 +1,408 @@
+
+    /**
+     * @var string
+     * @ORM\Column(type="string", length=255)
```
logo and url are nullable

gregurco commented on src/Entity/Job.php:
```diff
@@ -0,0 +1,408 @@
+     * @var string
+     * @ORM\Column(type="text")
+     */
+    private $how_to_apply;
```
also camelCase

eclimov committed:
- Unnecessary class name declaration removed from each entity
- All variable names transformed to camel case
- [jobs].logo and [jobs].url are now nullable by default

eclimov commented:
Fixed

gregurco commented:
@ksardedik :+1:
